From 245d08307fabcbc58e8ac0083005d0a6db12cbc7 Mon Sep 17 00:00:00 2001
From: Vaibhav Hiremath <hvaibhav@ti.com>
Date: Mon, 5 Mar 2012 20:22:47 +0530
Subject: [PATCH 0805/1092] ARM: OMAP3+: am33xx_clkdomain: Remove usb
 clkdomain entry

The USB peripheral doesn't have any seperate clock domain of its
own, so remove it.

Signed-off-by: Vaibhav Hiremath <hvaibhav@ti.com>
---
 arch/arm/mach-omap2/clockdomains33xx_data.c | 11 -----------
 1 file changed, 11 deletions(-)

diff --git a/arch/arm/mach-omap2/clockdomains33xx_data.c b/arch/arm/mach-omap2/clockdomains33xx_data.c
index cf7f858..7d653ce 100644
--- a/arch/arm/mach-omap2/clockdomains33xx_data.c
+++ b/arch/arm/mach-omap2/clockdomains33xx_data.c
@@ -202,16 +202,6 @@ static struct clockdomain l4_cefuse_am33xx_clkdm = {
 	.flags		= CLKDM_CAN_SWSUP,
 };
 
-static struct clockdomain wkup_usb_am33xx_clkdm = {
-	.name		= "wkup_usb_clkdm",
-	.pwrdm		= { .name = "wkup_pwrdm" },
-	.cm_inst	= AM33XX_CM_WKUP_MOD,
-	.prcm_partition	= AM33XX_PRM_PARTITION,
-	.clkdm_offs	= AM33XX_CM_CLKDCOLDO_DPLL_PER_OFFSET,
-	.clktrctrl_mask	= AM33XX_CLKTRCTRL_MASK,
-	.flags		= CLKDM_CAN_SWSUP,
-};
-
 static struct clockdomain *clockdomains_am33xx[] __initdata = {
 	&l4ls_am33xx_clkdm,
 	&l3s_am33xx_clkdm,
@@ -231,7 +221,6 @@ static struct clockdomain *clockdomains_am33xx[] __initdata = {
 	&gfx_l3_am33xx_clkdm,
 	&gfx_l4ls_gfx_am33xx_clkdm,
 	&l4_cefuse_am33xx_clkdm,
-	&wkup_usb_am33xx_clkdm,
 	NULL,
 };
 
-- 
1.7.11.2

